src/templates/app/components/loading.jsx

```diff
@@ -1,61 +1,21 @@
 import React               from 'react';
 import PropTypes           from 'prop-types';
-import {
```

Comment:
Same with this file, please create a new `loading` component. 🙏

Reply:
it won't affect the generic loader as i change the initial loading file, it is indeed the same name 👍